notebooks/WFC3/exception_report/wfc3_exception_report.ipynb

```diff
@@ -116,18 +116,21 @@
    "source": [
```

Comment:
<p>Hi! Thank you very much, @bjkuhn, for your work on the updates.</p><p>As a reader I would suggest we include a message for missing keywords. And also we can read header once per extension, which should make it more efficient.</p><p> </p><pre class="ql-syntax" spellcheck="false">for ext in range(1, numexts+1):
&nbsp; &nbsp; print("JIF Header Ext Number:", ext)
&nbsp; &nbsp; print('-'*80)
&nbsp; &nbsp; header = fits.getheader(jif_file, ext)
&nbsp; &nbsp; for keyword in keywords:
&nbsp; &nbsp; &nbsp; &nbsp; # try to display keyword because it may not be present&nbsp;
&nbsp; &nbsp; &nbsp; &nbsp; details = header.get(keyword)
&nbsp; &nbsp; &nbsp; &nbsp; if details is not None:
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; print(f'{details}')
&nbsp; &nbsp; &nbsp; &nbsp; else:
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; print(f'Keyword {keyword} not found in extension {ext}')
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/102/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>